packages/odyssey-storybook/src/components/odyssey-mui/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -86,43 +92,121 @@ type AutocompleteType = AutocompleteProps<
   boolean | undefined
 >;
 
-export const Default: StoryObj<AutocompleteType> = {
+const Template: StoryObj<AutocompleteType> = {
   render: function C(props) {
     return <Autocomplete {...props} options={stations} />;
   },
 };
```

Comment:
I think we can completely avoid this if we pass `stations` as a `default `options` prop to `storybookMeta`. Can we do that?

Reply:
@KevinGhadyani-Okta What are we trying to avoid, the Template? This same template is used in all the story variants and it follow other stories where we have templates setup this way. Can we deal with that refactor separately, if needed?